contracts/drops/oracle/DropsPriceOracleEthereum.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.13;
+
+import {Admin} from "../../abstract/Admin.sol";
+import {Initializable} from "../../abstract/Initializable.sol";
+
+import {IDropsPriceOracle} from "../interface/IDropsPriceOracle.sol";
+
+contract DropsPriceOracleEthereum is Admin, Initializable, IDropsPriceOracle {
+  /**
+   * @dev Constructor is left empty and init is used instead
+   */
+  constructor() {}
+
+  /**
+   * @notice Used internally to initialize the contract instead of through a constructor
+   * @dev This function is called by the deployer/factory when creating a contract
+   * @param initPayload abi encoded payload to use for contract initilaization
+   */
+  function init(bytes memory initPayload) external override returns (bytes4) {
+    require(!_isInitialized(), "HOLOGRAPH: already initialized");
+    assembly {
+      sstore(_adminSlot, origin())
+    }
+    _setInitialized();
+    return Initializable.init.selector;
+  }
+
+  function convertUsdToWei(uint256 usdAmount) external view returns (uint256 weiAmount) {
+    weiAmount = usdAmount;
```

Comment:
isn't this where an external call needs to be made to get the conversion?

Reply:
Yep, for now just returning same value since the actual external logic is not implemented.

Reply:
okay cool